src/types/index.d.ts

```diff
@@ -0,0 +1,11 @@
+export {}
+declare global {
+  namespace jest {
+    interface Matchers<R> {
+      toBeWithinRange(a: number, b: number): R
+    }
+    interface Matchers<R> {
+      toBeError(errorClass: Error, s: string): R
+    }
+  }
+}
```

Comment:
This declares `toBeError` for the whole repo, whereas it's define only for the file that it's implemented in.
Let's implement it for the whole repo as well in a setup file.

Reply:
I've removed this in favor of the package you linked above.